reference/tools/cmake/cmake.rst

```diff
@@ -35,7 +35,10 @@ when a package is being built directly by Conan (create, install)
             cmake.configure()
             cmake.build()
 
-
+``CMake`` ``configure()``, ``build()`` and ``install()`` methods have the ``subfolder`` parameter in case you need to generate the files
```

Comment:
I am not sure this is the intended use case?
The use case was having more than 1 CMakeLists.txt, in separate places, not as a single project?
I was thinking more an example like:
```python
def build(self):
     cmake1 = CMake(.., subfolder="folder1")
     cmake1.configure()
     cmake1.build()
     cmake2 = CMake(..., subfolder="folder2")
     cmake2.configure()
     cmake.build()
```